backend/models/categories.js

```diff
@@ -0,0 +1,17 @@
+const mongoose = require('mongoose');
```

Comment:
';' is not required in javascript. Please do enable format on save feature in vscode and remove ';' signs